src/test/java/lotto/model/LottoTest.java

```diff
@@ -0,0 +1,42 @@
+package lotto.model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static lotto.constants.ErrorMessage.INVALID_LOTTO_NUMBER_COUNT;
+import static lotto.constants.ErrorMessage.NOT_ALLOWED_DUPLICATED_NUMBERS;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class LottoTest {
+    @Test
+    void 로또_번호의_개수가_6개가_넘어가면_예외가_발생한다() {
+        assertThatThrownBy(() -> new Lotto(List.of(1, 2, 3, 4, 5, 6, 7)))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessageContaining(INVALID_LOTTO_NUMBER_COUNT);
+    }
+
+    @DisplayName("로또 번호에 중복된 숫자가 있으면 예외가 발생한다.")
```

Comment:
메서드 네이밍에 사용으로 해당 어노테이션은 중복된 거 같습니다.
```suggestion
    @DisplayName("로또 번호에 중복된 숫자가 있으면 예외가 발생한다.")
```

Reply:
아 DisplayName 말씀하시는 걸까요?
우테코에서 기본으로 제공해줬던 테스트인 거 같습니다 ㅎㅎ

이야기가 나와서 말씀드려보는 건데!

메서드 네이밍으로 충분한 설명이 되고 있다면
DisplayName 어노테이션은 사용하지 않아도 괜찮다고 생각하시는지 궁금합니다!

Reply:
아 착각했네요..! 하하 🤣 저는 DisplayName 어노테이션을 사용하지 않습니다. 궁극적인 이유는 한국어로 된 메서드 명을 작성할 때도 네이밍에 신경쓰게 되는데, 메서드명까지 고려하면 더욱 시간도 오래 걸린다고 생각했습니다.